nengo_gui/viz.py

```diff
@@ -1,419 +1,11 @@
-            config[self.model].size = (1.0, 1.0)
-
-        for k, v in self.locals.items():
-            if isinstance(v, nengo_gui.components.component.Template):
-                self.default_labels[v] = k
-        return config
-
-    def save_config(self, lazy=False, force=False):
-        if not force and not self.config_save_needed:
-            return
-
-        now_time = time.time()
-        if not force and lazy and self.config_save_time is not None:
-            if (now_time - self.config_save_time) < self.config_save_period:
-                return
-
-        with self.lock:
-            self.config_save_time = now_time
-            self.config_save_needed = False
-            try:
-                with open(self.config_name(), 'w') as f:
-                    f.write(self.config.dumps(uids=self.default_labels))
-            except IOError:
-                print("Could not save %s; permission denied" %
-                      self.config_name())
-
-    def modified_config(self):
-        self.config_save_needed = True
-
-    def get_label(self, obj, default_labels=None):
-        if default_labels is None:
-            default_labels = self.default_labels
-        label = obj.label
-        if label is None:
-            label = default_labels.get(obj, None)
-            if label is None:
-                print('ERROR finding label: %s' % obj)
-            else:
-                if '.' in label:
-                    label = label.rsplit('.', 1)[1]
-        if label is None:
-            label = repr(obj)
-        return label
-
-    def get_uid(self, obj, default_labels=None):
-        if default_labels is None:
-            default_labels = self.default_labels
-        uid = default_labels.get(obj, None)
-        if uid is None:
-            uid = repr(obj)
-        return uid
-
-    def start(self, port=8080, browser=True, password=None):
-        """Start the web server"""
-        print("Starting nengo_gui server at http://localhost:%d" % port)
-        if password is not None:
-            nengo_gui.server.Server.add_user('', password)
-            addr = ''
-        else:
-            addr = 'localhost'
-        nengo_gui.server.Server.start(self, port=port, browser=browser, addr=addr)
-
-    def prepare_server(self, viz, port=8080, browser=True):
-        return nengo_gui.server.Server.prepare_server(
-            self, port=port, browser=browser)
-
-    def begin_lifecycle(self, server):
-        nengo_gui.server.Server.begin_lifecycle(
-            server, interactive=self.interactive)
-
-    def create_sim(self):
-        """Create a new Simulator with this configuration"""
-        viz_sim = VizSim(self)
-        self.viz_sims.append(viz_sim)
-        return viz_sim
-
-    def remove_sim(self, viz_sim):
-        self.viz_sims.remove(viz_sim)
-
-    def count_sims(self):
-        return len(self.viz_sims)
+# dummy class so that old scripts still work, but get a deprecation message
```

Comment:
I can't imagine this will actually break anything for anyone, but it's a bit weird to me that calling this, I would get a different type than if I'd just made the `SimServer`. Could instead do this:
```python
def Viz(*args, **kwargs):
    warnings.warn("nengo_gui.Viz is deprecated. Use nengo_gui.SimServer instead.")
    return nengo_gui.SimServer(*args, **kwargs)
```